jobs/build/send-umb-messages/Jenkinsfile

```diff
@@ -25,33 +25,76 @@ node {
                     returnStdout: true,
                     script: "curl -s https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/${release}/latest",
                 ).trim()
-		try {
+                try {
                     previousRelease = readFile("${release}.current")
-		} catch (readex) {
-		    // The first time this job is ran and the first
-		    // time any new release is added the 'readFile'
-		    // won't find the file and will raise a
-		    // NoSuchFileException exception.
-		    touch file: "${release}.current"
-		    previousRelease = ""
-		}
+                } catch (readex) {
+                    // The first time this job is ran and the first
+                    // time any new release is added the 'readFile'
+                    // won't find the file and will raise a
+                    // NoSuchFileException exception.
+                    touch file: "${release}.current"
+                    previousRelease = ""
+                }
 
                 if ( latestRelease != previousRelease ) {
-		    currentBuild.displayName += "🆕: ${release} "
-		    currentBuild.description += "\n🆕: ${release}"
+                    currentBuild.displayName += "🆕: ${release} "
+                    currentBuild.description += "\n🆕: ${release}"
+
+                    def object = readJSON text: latestRelease
+
+                    def downloadURL = object.downloadURL
+                    def version = object.name
+                    def pullRef = object.pullSpec
+                    def digest = "MISSING_DIGEST_FIX_PLEASE"
+                    string repo = "somerepo"
+
+                    def ts = sh (
+                        returnStdout: true,
+                        script: 'date "+%Y-%m-%d\'T\'%H:%M:%S.%N\'Z\'"',
+                    ).trim()
+
+                    def msg = [
+                            artifact: [
+                                type: "container-image",
+                                repository: "${repo}",
+                                digest: "${digest}",
+                                issuer: "OpenShift Automated Release Team",
+                                pull_ref: "${pullRef}",
+                                scratch: false,
+                                id: "${repo}@${digest}",
+                            ],
+                            contact: [
+                                name: "Nightly-pipeline for OCP 4.x",
+                                team: "AOS QE",
+                                docs: "https://somewhere.com/user-documentation",
+                                email: "aos-qe@redhat.com",
+                                irc: "#aos-qe"
+                            ],
+                            decision: [
+                                result: "go",
+                                summary: "1 of 1 required tests passed",
+                                url: "https://datagrepper.somewhere.com/id?id=2018-21cc85d6"
```

Comment:
Do we possibly have any URL of the prow tests performed? This URL doesn't look alright.